backend/src/packages/chats/chatbot.service.ts

```diff
@@ -0,0 +1,24 @@
+import { type OpenAi } from '#libs/packages/open-ai/open-ai.js';
+
+import { NO_RESPONSE } from './libs/constants/constants.js';
+import { type OpenAiMessageGenerateRequestDto } from './libs/types/types.js';
+
+class ChatbotService {
+  private openAiService: OpenAi;
+  //TODO: add chat service
+  public constructor(openAi: OpenAi) {
+    this.openAiService = openAi;
+  }
+
+  public async generateReply({
+    content,
+  }: OpenAiMessageGenerateRequestDto): Promise<string | undefined> {
```

Comment:
```suggestion
  }: OpenAiMessageGenerateRequestDto): Promise<string | null> {
```
But can this function basically return undefined or null?